docs/farming-&-staking/farming/additional-guides/gpu-plotter.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Experimental GPU Plotter
+sidebar_position: 4
+description: Tips on GPU Plotter
+keywords:
+    - Plotter
+    - Farming
+    - GPU
+---
+
+The first test builds of the **GPU Plotter** are now available!!!
+
+[Download GPU Plotter](https://github.com/autonomys/subspace/actions/runs/10850781628)
+
+## Overview
+The GPU Plotter will eventually support most NVIDIA and AMD video cards. For now, only NVIDIA cards are supported.
+
+The download package includes both the node and farmer Advanced CLI executables, but you only need the farmer for plotting. It works for both standalone farming/plotting and farming clusters. Support for Space Acres will be added in the future.
```

Comment:
If we provide the direct links to only farmer, this sentence may be deleted, or rephrased to something like "You only need to use the test-build for the farmer, you can still use the general release for your node"